_data/gaming.yml

```diff
@@ -248,7 +248,7 @@ websites:
       doc: http://forums.ubi.com/showthread.php/1463703-Secure-your-account-with-2-Step-Verification
 
     - name: Wargaming
-      url: http://wargaming.net/
+      url: http://asia.wargaming.net/
```

Comment:
The old URL automatically redirects users to the correct server based on their location, for example, http://na,wargaming.net for users in North America. This is by design, and no change needs to be made to the previous URL.